t/porting/bincompat.t

```diff
@@ -1,19 +1,57 @@
 #!./perl -w
 use strict;
-require './test.pl';
+chdir ".." if -e "./test.pl";
+push @INC, "lib";
+require './t/test.pl';
 skip_all("Sorting order differs under EBCDIC") if $::IS_EBCDIC || $::IS_EBCDIC;
 
 use Config;
 
-plan(2);
+plan('no_plan');
```

Comment:
This is the old style "no plan" testing setup.

It's safer to not use a plan and call `done_testing();` after all of the tests are done.

Reply:
Hrn. I could have sworn that when i recently tried done_testing() in a test.pl based test file it didnt work. But now I see it in there. Sure, I'll use it, the only reason I didn't was because I thought I couldn't.

Reply:
Done. Thanks, TIL. :-)